target/linux/brcmiproc/bcm58800/base-files/lib/upgrade/platform.sh

```diff
@@ -0,0 +1,17 @@
+platform_check_image() {
+  # i know no way to verify the image
+  return 0;
+}
+
+platform_do_upgrade() {
+  sync
+  get_image "$1" | dd of=/dev/mmcblk0p5 bs=2M conv=fsync
+  sleep 1
+}
+
+platform_copy_config() {
+  mount /dev/mmcblk0p1 /mnt
+  cp -af "$CONF_TAR" /mnt/
+  sync
+  umount /mnt
```

Comment:
Still missing tab for indent in this file.

Reply:
Sorry, missed that the first time. However, have removed this file for now. Looks like having it is not mandatory. The samsung target and some of the other targets don't have it.